python/examples/simple_pipeline.py

```diff
@@ -56,6 +56,8 @@ def run(self, read_stream: ReadStream):
             print("TryPullOp: received {data}".format(data=data))
             time.sleep(0.5)
 
+def f() -> Stream[int]:
+    pass
```

Comment:
This code (and the corresponding `Stream` import) are not being used in the example. Please remove.

Reply:
I added it to check my annotations. Sure, will do.